src/wasm/wasm-debug.cpp

```diff
@@ -473,6 +473,10 @@ struct LocationUpdater {
   // Map start of line tables in the debug_line section to their new locations.
   std::map<BinaryLocation, BinaryLocation> debugLineMap;
 
+  // Map offset of location list entries in the debug_loc section to the base
+  // address of the compilation units referencing them.
+  std::map<uint64_t, BinaryLocation> debugLocMap;
```

Comment:
perhaps the name here could be expanded. e.g. `debugLineMap` before it is a map of old to new addresses for the same thing, but this is a different kind of map. Perhaps `debugLocToUnitMap`?

Reply:
done